packages/snaps-controllers/src/services/webview/WebViewMessageStream.test.ts

```diff
@@ -22,12 +22,12 @@ describe('WebViewMessageStream', () => {
     expect(await responsePromise).toBe(555);
 
     expect(mockWebViewA.injectJavaScript).toHaveBeenCalledWith(
-      `window.postMessage('eyJ0YXJnZXQiOiJiIiwiZGF0YSI6MTExfQ==')`,
+      `window.postMessage([123,34,116,97,114,103,101,116,34,58,34,98,34,44,34,100,97,116,97,34,58,34,83,89,78,34,125])`,
```

Comment:
Is the size increase not of concern? Does this work with very large Snap bundles etc.?